apps/laboratory/src/components/Ethers/EthersSendCallsTest.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button, Stack, Text, Spacer } from '@chakra-ui/react'
+import { useState } from 'react'
+import { useWeb3ModalAccount, useWeb3ModalProvider } from '@web3modal/ethers/react'
+import { EthereumProvider } from '@walletconnect/ethereum-provider'
+import { useChakraToast } from '../Toast'
+import { parseGwei, type Address } from 'viem'
+import { vitalikEthAddress } from '../../utils/DataUtil'
+import { BrowserProvider } from 'ethers'
+import { EIP_5792_RPC_METHODS, getAtomicBatchSupportedChainInfo } from '../../utils/EIP5792Utils'
+
+export function EthersSendCallsTest() {
+  const toast = useChakraToast()
+  const { address, chainId, isConnected } = useWeb3ModalAccount()
+  const { walletProvider } = useWeb3ModalProvider()
+  const [loading, setLoading] = useState(false)
+  async function onSendCalls() {
```

Comment:
Separate parts of the component with new line. Basically `state hooks/other hooks/functions/render` is good approach to separate them.

```suggestion

  async function onSendCalls() {
```

Reply:
Can you check if this done now?